CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
```

Comment:
I'd recommend running this file through a Markdown linter (VSCode has the [`markdownlint` extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)) as there's a few formatting tweaks I'd want to make before using this as our official `changelog` moving forward.

On that note, if we're going to make a CI pipeline for this repository (separate ticket, to be sure), we may also want to consider adding [this markdownlint action](https://github.com/avto-dev/markdown-lint) to our workflow, so we can this kind of feedback for free with any new PRs we submit.

Reply:
[Issue created](https://github.com/wayfair-incubator/oss-template/issues/10)

Reply:
Also handled manually in the latest commit, save for the "no repeated headlines" part for obvious reasons.

Reply:
sweet!